### PR TITLE
Linux fix - search lib and lib64

### PR DESCRIPTION
**Pull request by keflavich**
Will fix part of #7 and #12 which are duplicates (if it works...)

Testing now; do not merge.

### CODE REVIEW

**Comment by keflavich** [comment]
@astrofrog this now works on one test machine.  Could you review?  The order swap 2.7<->2.6 was because there was still a python2.6 directory that... seemed to be defunct; `io.py` would not import.  I don't think it should exist but it does.

File: setup_casapy_pip.py [hunk 1]
```diff
@@ -166,8 +173,11 @@ def write_casa_python_mac(pv="2.7"):
 
     casapy_path = os.path.dirname(os.path.dirname(get_casapy_path()))
 
+    assert pv
```
Comment (astrofrog) [inline setup_casapy_pip.py]: Remove?

File: setup_casapy_pip.py [hunk 2]
```diff
@@ -36,9 +36,12 @@ def make_executable(filename):
     st = os.stat(filename)
     os.chmod(filename, st.st_mode | stat.S_IEXEC)
 
+def find_osx_executable(fpath='/Applications/CASA.app/Contents/MacOS/casapy'):
+    if os.path.isfile(fpath) and os.access(fpath, os.X_OK):
+        return fpath
```
Comment (astrofrog) [inline setup_casapy_pip.py]: Should this be in this PR?

File: setup_casapy_pip.py [hunk 3]
```diff
@@ -229,6 +243,7 @@ def write_init(pv="2.7"):
 site.addsitedir("{site_packages}")
     """
 
+    assert pv
```
Comment (astrofrog) [inline setup_casapy_pip.py]: Remove?

File: setup_casapy_pip.py (hunk 2 above)
Comment (keflavich) [inline setup_casapy_pip.py]: no, I accidentally branched off the other pr.  easy to fix

**Comment by keflavich** [comment]
@astrofrog with those fixes, ok to merge?

**Comment by astrofrog** [comment]
Yep!

**Pull request merged**